src/Uno.UI.Composition/Composition/CompositionCapabilities.skia.cs

```diff
@@ -0,0 +1,18 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+
+namespace Windows.UI.Composition
+{
+	public partial class CompositionCapabilities
+	{
+		public bool AreEffectsSupported() => true;
```

Comment:
I wonder, does windows change this based on some performance (low battery) or user settings?

Reply:
Windows changes this based on minimum (safe) supported and maximum supported d3d hardware feature level and if it's running on software renderer or not, it also can be force enabled through registry.